emcc.py

```diff
@@ -1368,6 +1368,10 @@ def check(input_file):
       shared.Settings.OPT_LEVEL = opt_level
       shared.Settings.DEBUG_LEVEL = debug_level
 
+      if not shared.Settings.LEGALIZE_JS_FFI:
+        # Make sure is_wasm_only returns true
```

Comment:
this line of text is not needed, but please add more text to the assertion error message, explaining why (or pointing to settings.js if the details are there).